linkage-monitor/src/test/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitorTest.java

```diff
@@ -222,25 +222,14 @@ public void testBuildModelWithSnapshotBom_JdkVersionActivation()
   }
 
   @Test
-  public void testFindLocalArtifacts_currentDirectory() {
+  public void testFindLocalArtifacts() {
     // Current working directory of linkage-monitor should have one linkage monitor artifact
     ImmutableMap<String, String> localArtifacts =
-        LinkageMonitor.findLocalArtifacts(system, session, Paths.get("."));
+        LinkageMonitor.findLocalArtifacts(
+            system, session, Paths.get("src/test/resources/testproject"));
```

Comment:
Now this test only relies on the dummy directory containing the pom.xml files (added as part of this PR).